tutorials-and-examples/inference-servers/jetstream/pytorch/single-host-inference/README.md

```diff
@@ -0,0 +1,267 @@
+--checkpoint_path: Path to checkpoint
+Optional flags to add
+--quantize_weights (Default False): Checkpoint is quantized
+--quantize_kv_cache (Default False): Quantized kv cache
+```
+
+```
+kubectl apply -f deployment.yaml
+```
+
+### Verify the deployment
+```
+kubectl get deployment
+NAME               		READY   UP-TO-DATE   AVAILABLE   AGE
+jetstream-pytorch-server   2/2     2            2           ##s
+```
+
+View the HTTP server logs to check that the model has been loaded and compiled. It may take the server a few minutes to complete this operation.
+
+```
+kubectl logs deploy/jetstream-pytorch-server -f -c jetstream-http
+INFO:     Started server process [1]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
+```
+
+View the Jetstream Pytorch server logs and verify that the compilation is done.
+
+```
+kubectl logs deploy/jetstream-pytorch-server -f -c jetstream-pytorch-server
+Started jetstream_server....
+2024-04-12 04:33:37,128 - root - INFO - ---------Generate params 0 loaded.---------
+```
+
+## Serve the model
+
+```
+kubectl port-forward svc/jetstream-svc 8000:8000
+```
+
+Interact with the model via curl
+
+```
+curl --request POST \
+--header "Content-type: application/json" \
+-s \
+localhost:8000/generate \
+--data \
+'{
+    "prompt": "What are the top 5 programming languages",
+    "max_tokens": 200
+}'
+```
+
+The initial request can take several seconds to complete due to model warmup. The output is similar to the following:
+
+```
+{
+    "response": " for 2019?\nWhat are the top 5 programming languages for 2019? The top 5 programming languages for 2019 are Python, Java, JavaScript, C, and C++.\nWhat are the top 5 programming languages for 2019? The top 5 programming languages for 2019 are Python, Java, JavaScript, C, and C++. These languages are used in a variety of industries and are popular among developers.\nPython is a versatile language that can be used for web development, data analysis, and machine learning. It is easy to learn and has a large community of developers.\nJava is a popular language for enterprise applications and is used by many large companies. It is also used for mobile development and has a large community of developers.\nJavaScript is a popular language for web development and is used by many websites. It is also used for mobile development and has a"
+}
+```
+
+## Optionals
+
+### Interact with the Jetstream Pytorch server directly using gRPC
+
+The Jetstream HTTP server is great for initial testing and validating end-to-end requests and responses. If you would like to interact directly with the Jetstream Pytorch server directly for use cases such as [benchmarking](https://github.com/google/JetStream/tree/main/benchmarks), you can do so by following using Jetstream benchmarking setup and interacting with the Jetstream gRPC server at port 9000.
```

Comment:
```suggestion
The Jetstream HTTP server is great for initial testing and validating end-to-end requests and responses. In production use case, it's recommended to interact with the JetStream-Pytorch server directly for better throughput/latency and use the streaming decode feature on the JetStream grpc server.
```

Reply:
I guess want users to interact with the grpc server due to 1. better performance/throughput 2. have the streaming decode feature support

Reply:
Changed to suggested wording